cluster/log-dump/log-dump.sh

```diff
@@ -497,7 +497,7 @@ function dump_nodes() {
 # does not run on Windows nodes.
 #
 # Note: This step is O(#nodes^2) as we check if each node is present in the list of succeeded nodes.
-# Making it linear would add code complexity without much benefit (as it just takes ~1s for 5k nodes).
+# Making it linear would add code complexity without much benefit (as it takes ~1s for 5k nodes).
```

Comment:
This is talking about how long an automated process takes, not say, how difficult something is. 
The fact that it only takes ~1s for 5k nodes seems actually relevant.

I don't think any of the lines changed in this PR are discussing e.g. difficulty, "just" is being used as a synonym for "only" when discussing the limits of something (lack of additional functionality, limited time consumed computing something, etc.)

Reply:
I saw an issue in kubernetes/website, is about removing this kind of words, the issue number if 17070. I may misunderstand it.

Reply:
To be extra clear: 
- Thanks for doing this! Appreciate the help cleaning this sort of thing up ❤️ 
- I'm not sure this is _quite_ the set of changes we want, but this is only my suggestion and I'm going to ultimately defer to the wg/naming leadership (pinged in #wg-naming) 😅, I think some of the other unrelated changes should clearly be dropped though / moved to a different PR (https://github.com/kubernetes/kubernetes/pull/99548#discussion_r584255975)

Reply:
@BenTheElder Not to be a stick in the mud: I'd err on the side of keeping this change in :)

Reply:
[resolving this in favor]